examples/async_transcribe_media_from_url.js

```diff
@@ -11,9 +12,20 @@ const token = require('./config/config.json').access_token;
     console.log(`Account: ${account.email}`);
     console.log(`Credits remaining: ${account.balance_seconds} seconds`);
 
+    // Configure your source media url
+    // If authorization headers are needed to access the url they can be provided as an argument, e.g.
+    // var sourceConfig  {url: 'source url', auth_headers: {"Authorization": "Bearer <token>"}});
+    var sourceConfig = {url: 'https://www.rev.ai/FTC_Sample_1.mp3'};
```

Comment:
Can you explicitly set auth_headers None just to make sure it's clear the option exists